torch/__init__.py

```diff
@@ -778,3 +786,154 @@ def _register_device_module(device_type, module):
         raise RuntimeError("The runtime module of '{}' has already "
                            "been registered with '{}'".format(device_type, getattr(m, device_type)))
     setattr(m, device_type, module)
+
+
+class autocast(object):
+    r"""
+    Instances of :class:`autocast` serve as context managers or decorators that
```

Comment:
This documentation is a copy-paste from torch.cuda.amp, you should make it more generic and not hardcode "cuda" everywhere